test/TorSharp.Tests/TorSharpFetcherTests.cs

```diff
@@ -151,7 +151,7 @@ public static IEnumerable<object[]> Platforms
 
                     foreach (var architecture in Enum.GetValues(typeof(TorSharpArchitecture)).Cast<TorSharpArchitecture>())
                     {
-                        if (architecture == TorSharpArchitecture.Unknown)
+                        if (architecture == TorSharpArchitecture.Unknown || architecture.HasFlag(TorSharpArchitecture.Arm))
```

Comment:
We should test the Fetcher on in the test pass. We can't run the binaries (no GitHub Actions ARM agents available) but we can test the fetching.

Reply:
Oops, I see the new test class above. Nevermind!